imio/dms/mail/tests/test_subscribers.py

```
@@ -1636,3 +1639,57 @@ def test_cktemplate_moved(self):
+        self.portal.portal_workflow.doActionFor(request, "propose_to_approve")
+        self.assertEqual(ISignRequestApproval(request).current_nb, 0)
+
+    def test_sign_request_modified(self):
+        request, files = create_sign_request(self.portal, oid="sr-mod", nb_files=0)
+        request.signers = [{"number": 1, "signer": self.pf["bourgmestre"]["bourgmestre"].UID(),
+                            "approvings": [u"_themself_"], "editor": True}]
+        modified(request, Attributes(ISignRequestSigningBehavior, "ISignRequestSigningBehavior.signers"))
+        self.assertEqual(ISignRequestApproval(request).annot["approvers"], [["bourgmestre"]])
+
+    def test_sign_request_modified_duplicate_email(self):
+        request, files = create_sign_request(self.portal, oid="sr-dup", nb_files=0)
```

Ruff (0.15.21)

[warning] Unpacked variable `files` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)